### DISCUSSION TITLE
"Unhelpful" API pages

### DISCUSSION THREAD

**Discussion by Josh-Cena** [mdn/mdn-community#251] [Content] [answered]
There are some APIs that are purely for "plumbing" purposes of the language and likely do not interest developers. For example, virtually all new JS APIs (and all web APIs) have the [`@@toStringTag`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/toStringTag) property, which is solely used when stringifying the object—the string will be displayed as `[object JSON]`, etc.

However, of all objects that possess `@@toStringTag`, only `Intl[@@toStringTag]` and `Map.p[@@toStringTag]` happen to be documented, with very thin content that barely interests me. Maybe it's better if we remove API pages that aren't helpful for developing purposes? Can this principle be extended to some other properties?

**Comment by Rumyra** [comment, marked as answer]
As you've already done this @Josh-Cena I'm making a comment so we can mark it as answered :)
